admin-manual/installation/linux/ubuntu-trusty.rst

```diff
@@ -45,6 +48,13 @@ During the installation, you will be prompted to set a password for the default
 administrator user (root). We strongly recommend that you use a strong password
 and please write it down as you are going to need it later.
 
+.. warning::
+
+   If you are planning to use MySQL 5.7 please be aware that it has not been
+   fully tested yet and we know at least of `one bug <https://projects.artefactual.com/issues/9051>`__
+   that it has not been solved yet! MySQL 5.6 will work fine but it is not
```

Comment:
re: line 36 - in the Xenial instructions this is updated to recommending MySQL 5.6 - do we need to make changes for the Trusty docs too? What comes bundled, and is this why the difference?